Presentation.Web/app/components/global-admin/global-admin-misc.controller.ts

```diff
@@ -162,17 +160,17 @@
                 const nameAndEmail = `${$scope.selectedUser.text}, ${$scope.selectedUser.email}`;
 
                 if (confirm(`Er du sikker på, at du vil slette ${nameAndEmail}`)) {
-                    notify.addInfoMessage(`Sletter ${nameAndEmail}`);
+                    const message = notify.addInfoMessage(`Sletter ${nameAndEmail}`);
```

Comment:
using this approach it is possible to update the active notification in stead of stacking up